### PR TITLE
add a line ending to printed notes

### PR DESCRIPTION
**Pull request by topepo**
For some notes, `show_notes()` would not properly break the lines: 

``` r
library(tidymodels)
set.seed(1)
dat <- sim_classification(150, intercept = 15)
rs <- vfold_cv(dat)
fit_lr <- 
  logistic_reg() %>% 
  fit_resamples(
  class ~ .,
  rs,
  metrics = metric_set(accuracy, kap, recall, precision)
)
#> ! Fold01: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge
#> ! Fold02: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold03: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold04: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold05: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold06: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold07: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold08: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold09: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold10: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
show_notes(fit_lr)
#> unique notes:
#> ───────────────────────────────────
#> glm.fit: algorithm did not converge ───────────────────────────────────
#> glm.fit: algorithm did not converge, glm.fit: fitted probabilities numerically 0 or 1 occurred
```

<sup>Created on 2022-09-30 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

For this new version:

``` r
library(tidymodels)
set.seed(1)
dat <- sim_classification(150, intercept = 15)
rs <- vfold_cv(dat)
fit_lr <- 
  logistic_reg() %>% 
  fit_resamples(
  class ~ .,
  rs,
  metrics = metric_set(accuracy, kap, recall, precision)
)
#> ! Fold01: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge
#> ! Fold02: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold03: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold04: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold05: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold06: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold07: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold08: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold09: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
#> ! Fold10: preprocessor 1/1, model 1/1: glm.fit: algorithm did not converge, glm.fit: fitted probabilities numer...
show_notes(fit_lr)
#> unique notes:
#> ───────────────────────────────────
#> glm.fit: algorithm did not converge 
#>  ───────────────────────────────────
#> glm.fit: algorithm did not converge, glm.fit: fitted probabilities numerically 0 or 1 occurred
```

<sup>Created on 2022-09-30 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

### CODE REVIEW

File: tests/testthat/test-notes.R [hunk 1]
```diff
@@ -52,6 +53,19 @@ test_that("showing notes", {
...
+  # Get cli lines right
+  set.seed(1)
+  dat <- modeldata::sim_classification(150, intercept = 15)
+  rs <- rsample::vfold_cv(dat)
+  fit_lr <-
+    parsnip::logistic_reg() %>%
+    fit_resamples(
+      class ~ .,
+      rs,
+      metrics = yardstick::metric_set(accuracy, kap, recall, precision)
```
Comment (EmilHvitfeldt) [inline tests/testthat/test-notes.R:67]: Why not leave `metrics = NULL`? This way you would get a more minimal test, and you get a slightly larger output

File: tests/testthat/_snaps/notes.md [hunk 2]
```diff
@@ -79,3 +79,14 @@
...
 
+---
+
+    Code
+      show_notes(fit_lr)
+    Output
+      unique notes:
+      -----------------------------------
+      glm.fit: algorithm did not converge 
+       -----------------------------------
```
Comment (EmilHvitfeldt) [inline tests/testthat/_snaps/notes.md:90]: This might be a nitpick, but why does this line start with a space? 😅

Comment (topepo) [reply tests/testthat/_snaps/notes.md]: Lacking a `sep = ""` in `cat()`

File: tests/testthat/test-notes.R (hunk 1 above)
Comment (topepo) [reply tests/testthat/test-notes.R]: That was mostly there to help me trigger the warnings.

**Pull request merged**